### ISSUE TITLE
Page Should be scrolled towards the message area after product is added to cart

### ISSUE THREAD

**Issue by sudheer-gajjala** [magento/magento2#26098]
### Description (*)

Page Should be scrolled towards the message area after product is added to cart. So that the user can be acknowledged with the response message.

### Expected behavior (*)

Currently the page is not scrolled towards the messages section so, he will be missing out the specific information.

### Benefits

User can understand the event after he adds the product to cart.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @shikhamis11 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-29883` were created

**Issue Available**: @shikhamis11, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by krishprakash** [comment]
Hi @shikhamis11. I don't think this consider as an issue.
This is default magento behavior since magento 2.2.
Let me know your thoughts on this
Thanks.

**Comment by shikhamis11** [comment]
hi @krishprakash , Thank you for your feedback :) 
yes at first look it seems to be default behaviour and not an issue but if we go for user perspective this seems to be a issue thats why I confirmed it.

**Comment by krishprakash** [comment]
@shikhamis11  another use case as per user perspective:
 If user want to add to cart 2-3 products from same product row or after next row then every time user need to scroll down to that product row for add to cart that product.
As currently user can see the Add to cart button changed to added once click on it so user can know.

**Comment by shikhamis11** [comment]
@krishprakash 
there can be multiple cases , I will discuss about it and update here.

**Comment by sdzhepa** [comment]
Hello @sudheer-gajjala and @shikhamis11,  @krishprakash 

Thank you for contribution and collaboration!

First of all, I want to say that I agree with each of you in this thread:

- This is default Magento behavior that was made intentionally and currently it is expected behavior
- Also, it sounds like nice to have an improvement for user experience
- On the other hand, user scenario _"If user want to add to cart 2-3 products from same product row or after next row then every time user need to scroll down to that product row for add to cart that product."_ make sense because it could create a bad user experience for other groups of merchants/users


Anyway, such functional changes/improvements should be discussed and proved by Magento Product Owner(PO) who is responsible for this functionality. I will discuss it with the PO and share the result here but I will be able to do it only after 1st of January due to Christmas Holidays

**Comment by krishprakash** [comment]
#mm20in

**Comment by krishprakash** [comment]
@sdzhepa any suggestion for this improvement.
Thanks

**Comment by sdzhepa** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa, here is your [Magento instance](https://i-26098-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-26098-2-4-develop.instances.magento-community.engineering/admin_2684](https://i-26098-2-4-develop.instances.magento-community.engineering/admin_2684)
Login: `ffa537b9` Password: `5d681aedd2ee`
Instance will be terminated in up to 3 hours.

**Comment by shikhamis11** [comment]
hi @krishprakash @sdzhepa 
thanks for clarifications .. It was  intentionally made to improve user experience so I am closing this issue .
